example/timer/odp_timer_accuracy.c

```diff
@@ -814,9 +816,10 @@ static void print_stat(test_global_t *test_global)
 
 		for (int i = 0; i < (int)test_global->alloc_timers; i++) {
 			timer_ctx_t *t = &test_global->timer_ctx[i];
+			int64_t v = t->first_tmo_diff;
 
-			if (t->first_tmo_diff * t->first_tmo_diff > max * max) {
-				max = t->first_tmo_diff;
+			if (ABS(v) > ABS(max)) {
+				max = v;
```

Comment:
Please move usage of ABS() into the first patch. It makes code more readable and avoids unnecessary changes of these  same lines.